modules/ui/src/main/scala/lucuma/ui/hooks/package.scala

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.hooks
+
+export UseTheme.syntax.*
```

Comment:
What does this do?

Reply:
It allows you to import all syntax by means of `import lucuma.ui.hooks.*`. It is more useful when there are more hook definitions in the package.